### ISSUE TITLE
Accessibility issue: Contrast on link colors

### ISSUE THREAD

**Issue by chichiwang** [chichiwang/exercise-js#154]
The lighter blue color on links in the boilerplate UI fails accessibility for sufficient contrast on the white background. Fix this by using a darker blue for the links that has sufficient contrast with the white background.

Acceptance Criteria:
* Links are a darker blue and pass accessibility requirements for contrast

**Comment by chichiwang** [comment]
Task completed in PR #156
